src/main/resources/application.yaml

```diff
@@ -37,6 +37,10 @@ info:
   app:
     name: ${spring.application.name}
 
+idam:
+  url: ${IDAM_URL}
+  useStub: ${IDAM_USE_STUB:false}
```

Comment:
is the stub needed?

Reply:
only for people who want to run the app without having access to idam.

Reply:
Worth considering making an external stub, so that the draft store would unaware of stubbing. Without that stub being present, I'm ok with the code as it is.